src/Parser/DeathKnight/Unholy/Modules/Features/ScourgeStrikeEfficiency.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class ScourgeStrikeEfficiency extends Analyzer {
+  static dependencies = {
+    enemies: Enemies,
+    combantants: Combatants,
+  };
+
+  // used to track how many stacks a target has
+  targets = {};
+
+  totalScourgeStrikeCasts = 0;
+  scourgeStrikeCastsZeroWounds = 0;
+
+  on_byPlayer_applydebuffstack(event){
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.FESTERING_WOUND.id){
+      this.targets[event.targetID] = event.stack;
+    }
+  }
+
+  on_byPlayer_removedebuffstack(event){
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.FESTERING_WOUND.id){
+      this.targets[event.targetID] = event.stack;
+    }
+  }
+
+  on_byPlayer_removedebuff(event){
+    // keeps track of when the debuff drops off, useful for multitarget targets like DI
+    const spellId = event.ability.guid;
+    if(spellId === SPELLS.FESTERING_WOUND.id){
+      this.targets[event.targetID] = 0;
+    }
```

Comment:
Same thing with targetInstance here

Reply:
Can I simply just use targetInstance everywhere or do I have to use both?

Reply:
You have to use both.
If you simply check for instance, it could trigger off the same instance of a unit, despite them having different IDs 
Boss1 spawns addType1 x2 (so one is instance 1 and the other is instance 2) 
Boss1 spawns addType2 x2 (so one is instance 1 and the other is instance 2) 

Now imagine you hit the instance 2 of the addType1 and apply festering wound there, the analyzer would allow you to hit addType2 instance 2 and apply calculations there, even tho it wasn't the same mob.